dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/AbstractImageRegistry.java

```diff
@@ -276,6 +280,17 @@ public Tool refreshTool(final long toolId, final Long userId, final UserDAO user
         return updatedTool;
     }
 
+    /**
+     * Logs a refresh statement with the tool's descriptor language(s)
+     * @param tool                tool that is being refreshed
+     */
+    private void logToolRefresh(final Tool tool) {
+        List<String> descriptorTypes = tool.getDescriptorType();
+        for (String descriptorType : descriptorTypes) {
+            LOG.info("Refreshing " + descriptorType + " tool");
```

Comment:
I don't have the whole context for this, and I see the ticket says to only log the type, but since you're in here anyways why not log the actual tool as well? 

Just a thought, which may not be applicable.

Reply:
I don't see any harm in logging the tool name as well. However, if the intention is to create metrics that can be displayed on Grafana based on the tool names (ex: which tools are refreshed the most), it will be difficult unless we know exactly which tools we want to monitor.